### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by lilyhe123**
#57

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,15 +6,16 @@ WebLogic Monitoring Exporter
...
+You can find all the previous releases in [the releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).  
 Each release contains a getXXX.sh script. Download it and then run
 
 ```
-bash getXXX.sh <some-config-file>
+bash getXXX.sh <your-config-file>
 ```
 
-to download the release and install the specified default configuration.
+to download and build the web application `wls-exporter.war` with your configuration file.
```
Comment (russgold) [inline README.md]: This is not building the web application; it is simply downloading the WAR and adding the configuration file to it.

File: README.md [hunk 2]
```diff
@@ -6,15 +6,16 @@ WebLogic Monitoring Exporter
 The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
 It is deployed as a web application in a WebLogic Server (WLS) instance, version 12.2.1 or later, typically, in the instance from which you want to get metrics.
 
-## Building and configuring
+## Downloading the release
 
+You can find all the previous releases in the [releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).
```
Comment (rosemarymarano) [inline README.md]: You can find all the previous releases in the [releases page] -> You can find all the releases on the [releases page]

File: README.md [hunk 3]
```diff
@@ -6,15 +6,16 @@ WebLogic Monitoring Exporter
 The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
 It is deployed as a web application in a WebLogic Server (WLS) instance, version 12.2.1 or later, typically, in the instance from which you want to get metrics.
 
-## Building and configuring
+## Downloading the release
 
+You can find all the previous releases in the [releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).  
 Each release contains a getXXX.sh script. Download it and then run
```
Comment (rosemarymarano) [inline README.md]: Download it and then run -> (On a new line, put this text) To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script and then run:

File: README.md [hunk 4]
```diff
@@ -6,15 +6,16 @@ WebLogic Monitoring Exporter
...
 
 ```
-bash getXXX.sh <some-config-file>
+bash getXXX.sh <your-config-file>
 ```
 
-to download the release and install the specified default configuration.
+to download the web application `wls-exporter.war` and put your configuration file into the WAR.
```
Comment (rosemarymarano) [inline README.md]: to download the web application `wls-exporter.war` and put your configuration file into the WAR -> (remove this text from here).

File: samples/kubernetes/end2end/docs/existing-domain.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ We assume that you have already followed the [Manage Domains](https://oracle.git
 ### Installing the wls-exporter to the WebLogic domain
 You need to install the wls-exporter web application to the WebLogic domain.
 
-First, generate the wls-exporter WAR file following the [building from source](https://github.com/oracle/weblogic-monitoring-exporter#building-from-source) guide using the configuration file in this sample [exporter-config.yaml](../dashboard/exporter-config.yaml). Then deploy the WAR to the WebLogic servers and clusters using WLST or WDT.  
+First, get the wls-exporter WAR file following the [Downloading the release](https://github.com/oracle/weblogic-monitoring-exporter#downloading-the-release) guide using the configuration file in this sample [exporter-config.yaml](../dashboard/exporter-config.yaml). Then deploy the WAR to the WebLogic servers and clusters using WLST or WDT.
```
Comment (rosemarymarano) [inline samples/kubernetes/end2end/docs/existing-domain.md]: WebLogic servers -> WebLogic Servers

**Comment by russgold** [comment]
@lilyhe123 you know, you don't need to make a separate commit for every file; you can put all of the changes you make in a single commit.

**Pull request merged**